README.md

```diff
@@ -1,2 +1,223 @@
 # service-catalog-docs
 Documentation for the Sage Service Catalog
```

Comment:
If the docs aren't going to be maintained in github but in Confluence, are you going to rename this repo?

Reply:
Maybe?  It will depend on what @brucehoff suggests for SC-185.  The intention of adding the docs to this PR was mainly as a place holder, I really just copied and pasted it from the wiki and created the links to the scripts,  there are still many missing links.   My other intention was just to show how the docs would integrate with the files in this repo.